client/pom.xml

```diff
@@ -96,7 +100,7 @@
       <dependency>
         <groupId>org.jenkins-ci.main</groupId>
         <artifactId>remoting</artifactId>
-        <version>3.10.2</version>
+        <version>3.15</version>
```

Comment:
I see not reason in keeping the client compatible with Java 7 in the current state. There are many important stability fixes, and as a Remoting maintainer I have no plans to backport them. If needed, a new Java7-compatible release can be spinned from a branch